src/mock_bpa/bsl_mock_bpa_policy_config.c

```diff
@@ -528,11 +540,22 @@ static void mock_bpa_register_policy(const bsl_mock_policy_configuration_t polic
             break;
     }
 
+    BSL_HostEIDPattern_t eid_src_pat;
+    if (policy_ignore)
+    {
+        BSL_LOG_INFO("Creating src eid pattern ipn:0.0.0 - bundle should be ignored!");
+        eid_src_pat = mock_bpa_util_get_eid_pattern_from_text("ipn:0.0.0");
```

Comment:
This is actually a good question, an empty pattern should be a valid match-none case. Does the current API work to decode a text ""? I can update the decoder if it does not.

Reply:
Has this been tested with the empty string?

Reply:
I fixed this on this branch in 8c55edc